io.github.JakubMelka.Pdf4qt.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "io.github.JakubMelka.Pdf4qt",
+    "runtime": "org.kde.Sdk",
+    "runtime-version": "6.5",
+    "sdk": "org.kde.Sdk",
+    "command": "/app/bin/Pdf4QtViewerProfi",
+    "build-options": {
+	"build-args": [ "--share=network" ]
+	},
+    "finish-args": [
+        "--socket=x11",
```

Comment:
if it supports wayland then it's `fallback-x11`

Reply:
Fixed.